### PR TITLE
Implements the filter proposed for Filter Bar,

### PR DESCRIPTION
**Pull request by Camwyn**
to prevent Filter Bar classes on widget containers.

before: http://p.tri.be/MyDZq7
after: http://p.tri.be/roGu6x

Requires: https://github.com/moderntribe/events-filterbar/pull/359

🎫 [TEC-3614]

[TEC-3614]: https://moderntribe.atlassian.net/browse/TEC-3614

### CODE REVIEW

File: src/Tribe/Views/V2/Widgets/Widget_Abstract.php [hunk 1]
```diff
@@ -71,6 +71,9 @@ public function setup_view( $arguments ) {
 		$this->set_view( $view );
 
 		$this->enqueue_assets( $context, $view );
+
+		// Ensure widgets never get Filter Bar classes on their containers
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Widgets/Widget_Abstract.php:75]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 236c761d639b37ab5bbca9da6cbb7eaf</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

File: src/Tribe/Views/V2/Widgets/Widget_Abstract.php (hunk 1 above)
Comment (Camwyn) [reply src/Tribe/Views/V2/Widgets/Widget_Abstract.php:75]:
```suggestion
		// Ensure widgets never get Filter Bar classes on their containers.
```

**Review by lucatume — DISMISSED** [review]
Looks all fine, thanks.

**Review by lucatume — APPROVED** [review]

**Pull request merged**